src/Uno.UWP/UI/StringConversions.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// Partially ported
+// MUX Reference StringConversions.cpp, tag winui3/release/1.4.2
+
+using System;
+namespace Windows.UI;
+
+internal class StringConversions
+{
+	public static void
+		UnsignedFromHexString(
```

Comment:
If I understand what this is doing, I really would NOT have ported it over to Uno. It is not code that is easy to follow. Using C#'s built-in parsing methods can do most of this in one line of code. Again, I would not try to copy verbatim C++ and Windows methods for stuff like this... that said, as it could be used elsewhere, I guess I see the point in implementing the API at least.

Reply:
I agree, I only started porting it because the built-in methods don't have the exact semantics of the c++ implementation. But yes, I tried to find c# replacements whenever I could (like in `_wtoi`).

Reply:
I would keep the method `UnsignedFromHexString` and implement it using C#/.NET code.

This is a general problem in all the code ported so far. C++/WinUI has completely different culture/globalization mechanisms than .NET. However, it is better to use built-in .NET rules in cases like this. As a benefit, you also don't need to re-write methods like this -- just use .NET.